HLTrigger/Configuration/test/runIntegration.csh

```diff
@@ -28,44 +27,34 @@ endif
 foreach gtag ( $1 )
 
   if ( $gtag == DATA ) then
-    set flags  = ""
-    set infix  = hlt
+    set extraflags = "-x realData=1 -x globalTag=@"
   else
-    set flags  = --mc
-    set infix  = mc
+    set extraflags = "-x realData=0 -x globalTag=@"
   endif
 
   foreach table ( $tables )
 
     echo
     set name = HLT_Integration_${table}_${gtag}
-    touch  ${name}
+    touch ${name}
     rm -rf ${name}*
 
-    set config = `grep tableName OnLine_HLT_${table}.py | cut -f2 -d "'"`
-    if ($table == Fake) then
-      set basegt = auto:run1_${infix}_${table}
-    else if ( ($table == Fake1) || ($table == Fake2) || ($table == 2018) ) then
-      set basegt = auto:run2_${infix}_${table}
-    else
-      set basegt = auto:run3_${infix}_${table}
-    endif
-    set autogt = "--globaltag=${basegt}"
```

Comment:
Where is the globaltag now constructed and set for the integration tests?

Reply:
`globalTag=@` resolves to the GT hard-coded in [`OnLine_HLT_GRun`](https://raw.githubusercontent.com/cms-sw/cmssw/master/HLTrigger/Configuration/test/OnLine_HLT_GRun.py), the data or the MC one depending on the value of `realData`. This follows what is currently done in the [`addOnTests`](https://github.com/cms-sw/cmssw/blob/d38f357970f97327065d03b5d5f5a5a9b9cc10e7/Utilities/ReleaseScripts/scripts/addOnTests.py#L98).

Reply:
Ah great, removes this duplication then!